orangecontrib/timeseries/timeseries.py

```diff
@@ -29,6 +29,9 @@ def __init__(self, time_values):
             return
 
         deltas = list(np.sort(np.unique(np.diff(self.time_values))))
+        # in case several rows fall on the same datetime, remove the zero
+        if deltas[0] == 0:
+            deltas = deltas[1:]
```

Comment:
I don't think this is exactly right. What about a negative time delta? In this case 0 won't get deleted.
I propose to sort time_values first.
`deltas = list(np.sort(np.unique(np.diff(np.sort(self.time_values)))))`

Reply:
Are all timeseries not sorted by definition? Should they be?

Reply:
No, I am quite sure you can get 2020-01-01, 2020-01-03, 2020-01-02 in the data. We should make sure they are sorted.